src/cloud_tasks/cli.py

```
@@ -626,252 +470,933 @@ async def list_running_instances_cmd(args: argparse.Namespace, config: Config) -
+        task_db.insert_task(task["task_id"], task["data"], status="pending")
+        num_tasks += 1
+
+    logger.info(f"Loaded {num_tasks} tasks into database")
+
+    await task_queue.ensure_queue_ready()
+
+    logger.info(f"Enqueueing tasks to cloud queue '{queue_name}'...")
+    semaphore = asyncio.Semaphore(max_concurrent_queue_operations)
+    pending_tasks = set()
+
+    async def enqueue_task(task):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `enqueue_task`

Add return type annotation: `None`

(ANN202)